src/modelmanager.cpp

```diff
@@ -470,6 +470,25 @@ StatusCode downloadModels(std::shared_ptr<FileSystem>& fs, ModelConfig& config,
     return StatusCode::OK;
 }
 
+Status ModelManager::cleanupModelTmpFiles(ModelConfig& config, std::shared_ptr<model_versions_t>& versionsToCleanup) {
```

Comment:
Why do we pass shared pointer of std::vector just to check its size? We only execute instructions in this method if its size is 0. Maybe just not call this method at all if vector is empty in first place?

Reply:
Good catch.

Reply:
We decided unit testing cloud storage is problematic and would require to much effort compared to gain - so we will not do that.

Reply:
We no longer maintain the status of non blocking failed commands.